community/en/multi_class_classification/food_classifier.ipynb

```diff
@@ -0,0 +1,1165 @@
+        "* Validation accuracy has been on the higher side than training accuracy for many epochs\n",
+        "* This could be for several reasons:\n",
+        "  * We used a pretrained model trained on ImageNet which contains data from a variety of classes\n",
+        "  * Using dropout can lead to a higher validation accuracy\n",
+        "* The best model saved has a Top-1 validation accuracy of 93%\n",
+        " \n"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "qjeWMHrCwSoS"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# Predicting classes for new images from internet using the best trained model"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "XBb-sj73pNc7",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%%time\n",
+        "# Loading the best saved model to make predictions\n",
+        "\n",
+        "K.clear_session()\n",
+        "model_best = load_model('bestmodel_3class.hdf5',compile = False)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "nYTIzLhiRNKt"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "* **Setting compile=False and clearing the session leads to faster loading of the saved model**\n",
+        "* **Withouth the above addiitons, model loading was taking more than a minute!**"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "5MIBtyj1pFoK",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "def predict_class(model, images, show = True):\n",
+        "  for img in images:\n",
+        "    img = image.load_img(img, target_size=(299, 299))\n",
+        "    img = image.img_to_array(img)                    \n",
+        "    img = np.expand_dims(img, axis=0)         \n",
+        "    img = preprocess_input(img)                                      \n",
+        "\n",
+        "    pred = model.predict(img)\n",
+        "    index = np.argmax(pred)\n",
+        "    food_list.sort()\n",
+        "    pred_value = food_list[index]\n",
+        "    #print(pred)\n",
+        "    if show:\n",
+        "        plt.imshow(img[0])                           \n",
+        "        plt.axis('off')\n",
+        "        plt.title(pred_value)\n",
+        "        plt.show()\n",
+        "  "
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "uzLVocClxD0f",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Make a list of images and test the trained model\n",
+        "images = []\n",
+        "imagepath = '/content/examples/community/en/multi_class_classification/images/'\n",
```

Comment:
This cell breaks because the notebook hasn't been committed, yet: https://colab.research.google.com/github/theimgclist/examples/blob/MultiClassTF2.0/community/en/multi_class_classification/food_classifier.ipynb

Additionally, we don't know the copyright on these images and can't include them in the repo. Can you rework this section to only use the download images and remove the /images directory from ths PR? Thanks

Reply:
Thanks for the code cleanup @lamberta. Couldn't get back to working on this as I was caught up in few things.
Sure, I will remove those from repo. I initially used wget utility to download those images directly in the notebook. Will that work? Also, in one of your previous comments you mentioned, "I'm uncomfortable with making the user download and run external files." So I thought I could download images and add them to the repo. How about using Colab's file upload to add a new image for testing?

Please confirm this change and I will go ahead and make the changes. Thanks!